lib/redux/client/client_actions.dart

```diff
@@ -261,7 +262,8 @@ class FilterClientsByCustom2 implements PersistUI {
 }
 
 void handleClientAction(
-    BuildContext context, ClientEntity client, EntityAction action) {
+    BuildContext context, ClientEntity client, EntityAction action,
+    {bool multiselect = false, bool isMultiselectLast = false}) {
```

Comment:
Do we need to pass in multiselect and isMultiselectLast, this function has access to the app state so should be able to determine this?